Makefile

```diff
@@ -176,72 +190,62 @@ cycle: ## Triggers kusk-gateway-manager deployment rollout restart to pick up th
 .PHONY: cycle-envoy
 cycle-envoy: ## Triggers all Envoy pods in the cluster to restart
 	kubectl rollout restart deployment/kgw-envoy-default -n default
-	-kubectl rollout restart deployment/kgw-envoy-testing -n testing
+	kubectl rollout restart deployment/kgw-envoy-testing -n testing || echo 'rollout restart failed'
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-.PHONY: controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0)
+$(KUSTOMIZE): ## Download kustomize locally if necessary.
+# Old way/version.
```

Comment:
If you've tested that kustomize v4 works as expected, this comment can be removed

Reply:
The `main` branch now uses `sigs.k8s.io/kustomize/kustomize/v4@v4.5.4` so I guess this is no longer relevant. Marking as resolved.